src/earn/EarnEnterAmount.tsx

```diff
@@ -276,20 +241,21 @@ function EarnEnterAmount({ route }: Props) {
         transactionToken,
         feeCurrencies
       )
-    }, FETCH_UPDATED_TRANSACTIONS_DEBOUNCE_TIME)
+    }, FETCH_UPDATED_TRANSACTIONS_DEBOUNCE_TIME_MS)
     return () => clearTimeout(debouncedRefreshTransactions)
-  }, [tokenAmount, mode, transactionToken, feeCurrencies])
+  }, [processedAmounts.token.bignum, mode, transactionToken, feeCurrencies])
```

Comment:
```suggestion
  }, [processedAmounts.token.bignum.toString(), mode, transactionToken, feeCurrencies])
```
maybe makes the memoisation more reliable